### DISCUSSION TITLE
GitHub Copilot VSCode extension not usable anymore -> "extension host error"

### DISCUSSION THREAD

**Discussion by tunayokumus** [community/community#49987] [Copilot Conversations]
### Select Topic Area

Bug

### Body

- VS Code Version: 1.76.1
- OS Version: Windows NT x64 10.0.19044 (WSL2 backend)

Steps to Reproduce:

1. Use VSCode on Windows with WSL2 backend or on a Linux-OS.
2. Have the GitHub Copilot extension enabled and active.
3. Wait for very frequent crashes. (Frequent extension host termination warning)

It seems there are multiple issues with the title "extension host terminated..." reported in the last week. It is often reported that this happens on Linux OS / WSL2. The issue is resolved once the GitHub Copilot extension is deactivated/disabled.

https://github.com/microsoft/vscode/issues/177017
https://github.com/microsoft/vscode/issues/176562
https://github.com/microsoft/vscode/issues/176957
https://github.com/microsoft/vscode/issues/176868
https://github.com/microsoft/vscode/issues/176841
https://github.com/microsoft/vscode/issues/176733
https://github.com/microsoft/vscode/issues/176716
https://github.com/microsoft/vscode/issues/176624

**Comment by Relaxe111** [comment]
same on mac
